nhost/compose/wrapper.go

```diff
@@ -70,6 +72,18 @@ func (w Wrapper) ensureFoldersExistForDockerVolumes(workdir, gitBranch string) e
 		}
 	}
 
+	// write pg_hba_local.conf
+	hbaFile := filepath.Join(dotNhostFolder, DbDataDirGitBranchScopedPath(gitBranch, "pg_hba_local.conf"))
+	hbaFileContent := `
+  local all all trust
+	host all all all trust
```

Comment:
<img width="553" alt="image" src="https://user-images.githubusercontent.com/48279149/188331151-ffa54a7d-4953-4414-acea-5d86f08f823d.png">

let's use either spaces or tabs, but not a mix